processor/k8sattributesprocessor/options.go

```diff
@@ -40,6 +40,7 @@ const (
 	metadataNode       = "node"
 	// Will be removed when new fields get merged to https://github.com/open-telemetry/opentelemetry-collector/blob/main/model/semconv/opentelemetry.go
 	metadataPodStartTime = "k8s.pod.start_time"
+	specPodHostName      = "k8s.pod.hostname"
```

Comment:
what's the reason this is prefixed w/ `spec` and not `metadata`?

Reply:
cause the hostname is not in pod metadata section. It is actually a field in pod spec section.

Reply:
@codeboten do you still have concerns about this one? thx

Reply:
all good, thanks for clarifying